demo/src/routes/docs/[framework]/daisyUI/slider/+layout.server.ts

```diff
@@ -0,0 +1,17 @@
+import {daisyUIMetadata} from '$lib/components-metadata';
+
+export const load = async () => {
+	return {
+		...daisyUIMetadata.Slider,
+		prev: {
```

Comment:
You also need to update the prev tag of +layout.server.ts file of Accordion to target the slider.